### PR TITLE
Update CHANGELOG with #3063

### PR DESCRIPTION
**Pull request by mhils**
As the title says. :smiley:

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -18,6 +18,8 @@ Changelog
   and
   :meth:`~cryptography.hazmat.primitives.asymmetric.ec.EllipticCurvePublicKey.verify`
   methods to ECDSA keys.
+* Switched back to the older callback model on Python 3.5 in order to mitigate the locking
```
Comment (reaperhulk) [inline CHANGELOG.rst]: This fails the docs job because the line is >79 chars long

**Comment by reaperhulk** [comment]
Jenkins, ok to test

**Comment by mhils** [comment]
Line-length is fixed, not sure what Jenkins is complaining about now. Let's see how far my permissions go here :stuck_out_tongue: 

Jenkins, retest this please.

**Comment by reaperhulk** [comment]
Jenkins is currently on a version that leaks memory (but later versions have some other issues so I haven't upgraded) so it ran out of permgen space :) We're back up now

**Comment by alex** [comment]
Jenkins, retest this please.

**Pull request merged**